pkg/translator/jaeger/traces_to_jaegerproto.go

```diff
@@ -257,34 +235,23 @@ func getJaegerProtoSpanTags(span pdata.Span, instrumentationLibrary pdata.Instru
 	return tags
 }
 
-func traceIDToJaegerProto(traceID pdata.TraceID) (model.TraceID, error) {
+func traceIDToJaegerProto(traceID pdata.TraceID) model.TraceID {
 	traceIDHigh, traceIDLow := idutils.TraceIDToUInt64Pair(traceID)
-	if traceIDLow == 0 && traceIDHigh == 0 {
```

Comment:
Another option is to keep this and handle the error case in the caller, returning a nil span in case of error.

Reply:
I think this validation should be an explicit step configured by the user and not done in any way automatically.